test/integration/auth/podsecurity_test.go

```diff
@@ -75,7 +75,11 @@ func TestPodSecurity(t *testing.T) {
 func TestPodSecurityGAOnly(t *testing.T) {
 	// Disable all alpha and beta features
 	for k, v := range utilfeature.DefaultFeatureGate.DeepCopy().GetAll() {
-		if v.PreRelease == featuregate.Alpha || v.PreRelease == featuregate.Beta {
+		if k == "AllAlpha" || k == "AllBeta" {
```

Comment:
also considered just swapping out the entire `utilfeature.DefaultFeatureGate` but that cannot be done atomically, not sure if that is a consideration in the test environment

Reply:
this is fine for now, I'll open a follow up to make SetFeatureGateDuringTest safe no matter what you throw at it